includes/Modules/Tag_Manager.php

```diff
@@ -444,17 +420,24 @@ protected function get_datapoint_services() {
 	 * @return array Map of datapoints to their definitions.
 	 */
 	protected function get_datapoint_definitions() {
-		$map = parent::get_datapoint_definitions();
-
-		$map['POST:create-container'] = array_merge(
-			$map['POST:create-container'],
-			array(
+		return array(
+			'GET:accounts'               => array( 'service' => 'tagmanager' ),
+			'GET:accounts-containers'    => array( 'service' => 'tagmanager' ),
```

Comment:
These first two lines should come after `POST:account-id`.

Reply:
Updated.